src/types.ts

```diff
@@ -37,6 +38,7 @@ export interface MergedClockWeatherCardConfig extends LovelaceCardConfig {
   date_pattern: string;
   hide_today_section: boolean;
   hide_forecast_section: boolean;
+  forecast_hourly: boolean;
```

Comment:
wouldn't `hourly_forecast` make more sense?

Reply:
changed it accordingly